cttportfolio.css

```
@@ -197,15 +264,14 @@ h2{
 	flex-direction:column;
 	justify-content:center;
 	margin:auto 0;
-	padding-top:40px;
+	padding-top:10px
 }
 .am_text > p{
 	font-size:2.0rem;
 }
 .am_text > *{
```

```suggestion
.am_text > *{
```
why do you use these operators when you hav access to the direct html file?

---

this targets all the child elements of the selected class. is that what you were asking?